src/app/containers/Video/index.jsx

```diff
@@ -51,6 +51,24 @@ const VideoContainer = ({ blocks }) => {
     },
   ];
 
+  // prettier-ignore
```

Comment:
we need to have prettier-ignore here or prettier will add trailing `,`s which aren't valid json and break the mediaplayer when passed in like that